cli/run_test.go

```diff
@@ -94,9 +95,20 @@ func TestRunErrorHandler(t *testing.T) {
 }
 
 func TestRunContext(t *testing.T) {
+	testFlagName := "test-flag"
```

Comment:
Fine for this PR, but FYI you can have const blocks in functions as well, so this can be rewritten as:

```go
const(
    testFlagName = "test-flag"
    testFlagValue = "foobar"
)
```